mx.sulong/suite.py

```diff
@@ -357,11 +357,25 @@
       "license" : "BSD-new",
     },
 
+    "com.oracle.truffle.llvm.assemblerX86" : {
```

Comment:
Maybe we can rename it to `com.oracle.truffle.llvm.asm.amd64`? Then it would be consistent with `com.oracle.graal.asm.amd64`.